.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[alias]
+ci = "run --package ci"
```

Comment:
nit: missing trailing newline

Reply:
If trailing newlines are important, we should add a check to CI. 

> I'd seen that but hadn't read it fully - in that case I'd argue we just shouldn't have a `rustfmt.toml` checked in at the root, and instead have it somewhere else. Then `cargo xtask fmt` would be `cargo +nightly fmt -- config-path <path_to_rusfmt_toml>`

AFAIK this would break @cart's desired experience. He uses nightly, and wants those options to take affect in his environment.

Reply:
In git's view the newline is part of the current line and as such adding a missing newline to insert a line below the final line will cause the old final line to be considered as changed.

Reply:
Thanks for pointing out, that's good to know!  I encourage you to look for a way to integrate that check into one of the linters' configs if it bothers you.

In general, if you use the `Insert a suggestion` feature for nit-level concerns I'd be happy to hit the "accept change" button to commit it with you as co-author. Otherwise, I typically don't worry about nits unless I've got to modify the code in question for some other reason anyway.  I just wanted to make sure that you knew that I wasn't trying to ignore you or disagree with you.

Reply:
I was looking at this PR again tonight, so I went ahead and added that newline. 😄 

By the way, I'm baffled as to why there wasn't a trailing newline in the first place. Take a look at my "Even Better Toml" config:

<img width="505" alt="image" src="https://user-images.githubusercontent.com/5838512/109409028-5c6c2a00-794c-11eb-8110-c0eb8326643d.png">

I went and checked to see if [super-linter](https://github.com/github/super-linter) (which we're using for markdown linting) has a toml linter that I could add to CI, and it doesn't.  😞  Guess I'll just keep a sharper eye out for this happening in the future.